vendor/users/config/01-acls.server.config.js

```diff
@@ -173,7 +176,7 @@ module.exports = (app, db, config) => {
         }
 
         // Add the isAllowed middleware to all subroutes
-        allMiddlwares.unshift(isAllowed(iam));
```

Comment:
`iam` is an empty mongoose model declared on line **21**, also `isAllowed` function was accepting no parameters but was receiving the `iam` param. which is kind of weird.